crates/utils/src/error.rs

```diff
@@ -33,7 +33,7 @@ pub enum LemmyErrorType {
   PictrsApiKeyNotProvided,
   NoContentTypeHeader,
   NotAnImageType,
-  InvalidImageUpload,
+  InvalidImageUpload(String),
```

Comment:
We have a couple of other `Pictrs` related errors above, so this should probably be called `PictrsImageUploadError(String)`